ui/tooltip.js

```diff
@@ -1,9 +1,12 @@
+import Quill from '../core/quill';
+
 class Tooltip {
   constructor(quill, boundsContainer) {
     this.quill = quill;
     this.boundsContainer = boundsContainer || document.body;
     this.root = quill.addContainer('ql-tooltip');
-    this.root.innerHTML = this.constructor.TEMPLATE;
+    // Security: Blessed HTML is provided by the theme.
+    this.root.innerHTML = Quill.import('core/security').blessHTML(this.constructor.TEMPLATE);
```

Comment:
Consider moving this blessHTML class into the individual themes. Although far fetched, there isn't anything preventing apps from - say - allowing users to customize their Quill theme through a setting, which would lead to XSS.